src/rgw/rgw_bucket.cc

```diff
@@ -2255,7 +2255,7 @@ class RGWBucketInstanceMetadataHandler : public RGWMetadataHandler {
       bci.info.bucket.tenant = tenant_name;
       ret = store->select_bucket_location_by_rule(bci.info.placement_rule, &rule_info);
       if (ret < 0) {
-        ldout(store->ctx(), 0) << "ERROR: select_bucket_placement() returned " << ret << dendl;
+        ldout(store->ctx(), 0) << "ERROR: select_bucket_location_by_rule() returned " << ret << dendl;
```

Comment:
OK, I think we'd better drop this, because we want to rename this (like https://github.com/ceph/ceph/pull/15797/files#diff-c30965955342b98393b73be699f4e355R5890 ) finally.